File: DATAMATIKER/4/Mobile/HelloWorld/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import android.support.v7.app.AppCompatActivity;
import android.util.Log;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d("Hey", "LOL");
        Log.w("Big Dig", "LOGMSG");
        Log.e("RADIO", "NOT WORKING");
        setContentView(R.layout.activity_main);
    }
}

File: DATAMATIKER/4/MobileX/ClickApp/ClickApp/MainActivity.cs
using Android.App;
using Android.OS;
using Android.Support.V7.App;
using Android.Runtime;
using Android.Widget;

namespace ClickApp
{
    [Activity(Label = "@string/AppHeadline", Theme = "@style/AppTheme", MainLauncher = true)]
    public class MainActivity : AppCompatActivity
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.activity_main);

            Button button1 = FindViewById<Button>(Resource.Id.myButton);
            button1.Click += (sender, e) =>
            {
                StartActivity(typeof(HelloActivity));
            };
        }
    }
}